supabase/migrations/20260806202503_builder_capacity_events.sql

```
@@ -0,0 +1,72 @@
+ALTER TABLE public.build_requests
+  ADD COLUMN IF NOT EXISTS started_at timestamp with time zone,
+  ADD COLUMN IF NOT EXISTS completed_at timestamp with time zone;
+
+COMMENT ON COLUMN public.build_requests.started_at IS
+  'Builder-reported job start (UTC). Used for capacity reconstruction.';
+COMMENT ON COLUMN public.build_requests.completed_at IS
+  'Builder-reported job completion (UTC). Used for capacity reconstruction.';
+
+CREATE INDEX IF NOT EXISTS idx_build_requests_started_at
+  ON public.build_requests USING btree (started_at)
+  WHERE started_at IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)